src/redis-pubsub.ts

```diff
@@ -89,12 +98,16 @@ export class RedisPubSub implements PubSubEngine {
   public subscribe<T = any>(
     trigger: string,
     onMessage: OnMessage<T>,
-    options: unknown = {},
+    options: SubscribeOptions = {},
   ): Promise<number> {
 
     const triggerName: string = this.triggerTransform(trigger, options);
     const id = this.currentSubscriptionId++;
-    this.subscriptionMap[id] = [triggerName, onMessage];
+    if (options['includeChannel']) {
+	this.subscriptionMap[id] = [triggerName, onMessage, true];
+    } else {
+	this.subscriptionMap[id] = [triggerName, onMessage];
```

Comment:
This appeared to be a performance or memory optimization.  So, I purposefully tried to not change any existing memory requirements if you don't specify `includeChannel` at all.